tests/test_wfsim.py

```diff
@@ -4,27 +4,16 @@
 import straxen
 import wfsim
 
-strax.mailbox.Mailbox.DEFAULT_TIMEOUT = 60
-
 run_id = '010000'
 
-
 def test_sim_1T():
     """Test the 1T simulator (should always work with the publicly available files)"""
     with tempfile.TemporaryDirectory() as tempdir:
         testing_config_1T = dict(
             hev_gain_model=('to_pe_constant', 0.0085),
             gain_model=('to_pe_constant', 0.0085)
         )
-        st = strax.Context(
-            storage=tempdir,
-            config=dict(
-                nchunk=1, event_rate=1, chunk_size=10,
-                detector='XENON1T',
-                fax_config='https://raw.githubusercontent.com/XENONnT/strax_auxiliary_files/7ba4875b52162cbbe9284faf66a6b9193a254a30/sim_files/fax_config_1t.json',  # noqa
-                **straxen.contexts.x1t_common_config),
-            **straxen.contexts.common_opts)
-        st.register(wfsim.RawRecordsFromFax1T)
+        st = straxen.contexts.xenon1t_simulation(output_folder=tempdir)
```

Comment:
The function will now use the private `fax_config` file defined in https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L329-L340 or? The docstring outlines that it should work with publicly available files.

Reply:
I think the answer is given here:
https://github.com/XENONnT/WFSim/pull/88